src/util.ts

```diff
@@ -38,6 +39,17 @@ function copyElementAttributes(destinationElement: Element, sourceElement: Eleme
   }
 }
 
+function copyScriptAttributes(destinationElement: HTMLScriptElement, sourceElement: HTMLScriptElement) {
```

Comment:
should we add a test for createScriptURL too?

Reply:
Good catch, I'll get one added for that.